src/time-picker/time-display.jsx

```diff
@@ -162,26 +164,27 @@ const TimeDisplay = React.createClass({
     }
 
     return (
-      <div {...other} style={this.prepareStyles(styles.root)}>
-        <div style={this.prepareStyles(styles.box)}>
-          <div style={this.prepareStyles(styles.text)}>
-            <div style={this.prepareStyles(styles.affix)} />
-            <div style={this.prepareStyles(styles.time)}>
+      <div {...other} style={prepareStyles(styles.root)}>
+        <div style={prepareStyles(styles.box)}>
+          <div style={prepareStyles(styles.text)}>
+            <div style={prepareStyles(Object.assign({}, styles.affix))} />
```

Comment:
Yeah, I'm not sure how we feel about that but that's exactly the reason. `prepareStyles()` doesn't create a new object like it used to, I don't know whether it should or not.